### PR TITLE
chore: Update Package.swift for Cap6 release

### PR DESCRIPTION
**Pull request by markemer**

### CODE REVIEW

**Review by giralte-ionic — APPROVED** [review]

File: keyboard/Package.swift [hunk 1]
```diff
@@ -7,18 +7,18 @@ let package = Package(
     platforms: [.iOS(.v13)],
     products: [
         .library(
-            name: "KeyboardPlugin",
+            name: "CapacitorKeyboardPlugin",
```
Comment (jcesarmobile) [inline keyboard/Package.swift:10]:
```suggestion
            name: "CapacitorKeyboard",
```

File: status-bar/Package.swift [hunk 2]
```diff
@@ -7,18 +7,18 @@ let package = Package(
     platforms: [.iOS(.v13)],
     products: [
         .library(
-            name: "StatusBarPlugin",
+            name: "CapacitorStatusBarPlugin",
```
Comment (jcesarmobile) [inline status-bar/Package.swift:10]:
```suggestion
            name: "CapacitorStatusBar",
```

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
keyboard and status bar have incorrect names

**Comment by markemer** [comment]
@jcesarmobile I got those two, I also changed the package name since that will also cause problems. Let me know if you find more - I think I got them all.

**Review by theproducer — APPROVED** [review]

**Review by jcesarmobile — APPROVED** [review]

**Pull request merged**